### PR TITLE
doc/user: complete and polish release notes for v0.20.0

### PR DESCRIPTION
**Pull request by benesch**
### Motivation

* Weekly release notes PR.

### Checklist

- [x] This PR has adequate test coverage / QA involvement has been duly considered.
- [x] This PR adds a release note for any [user-facing behavior changes](https://github.com/MaterializeInc/materialize/blob/main/doc/user/content/release-notes.md#what-changes-require-a-release-note).

### CODE REVIEW

**Comment by benesch** [comment]
@morsapaes I'm going to eagerly merge this to avoid conflicts, but please do take a look and make whatever edits you see fit.

File: doc/user/layouts/partials/specifying-aws-credentials.html [hunk 1]
```diff
@@ -9,10 +9,12 @@
    [`AWS_WEB_IDENTITY_TOKEN_FILE`][web-identity], and [many more][env-vars].
 3. `credential_process` command in the AWS config file, by default located at `~/.aws/config`.
 4. AWS credentials file and profile files. By default located at `~/.aws/config` and
-   `~/.aws/credentials` , respectively.
+   `~/.aws/credentials`, respectively.
 5. The [IAM instance profile][instance-profile] provided by the Instance Metadata Service. This
    will only work if running on an EC2 instance with an instance profile.
 
+*Changed in v0.10.0:* Materialize.
```
Comment (morsapaes) [inline doc/user/layouts/partials/specifying-aws-credentials.html:16]: Is this...intended?

File: doc/user/content/release-notes.md [hunk 2]
```diff
@@ -107,80 +107,98 @@ These changes are present in [unstable builds](/versions/#unstable-builds) and
...
+{{</ comment >}}
+
+{{% version-header v0.20.0 %}}
+
+- **Breaking change.** In Kafka sources and sinks, do not default the SSL
+  parameters for the Confluent Schema Registry to the SSL parameters for the
+  Kafka broker.
+
+  SSL parameters for the Confluent Schema Registry must now always be provided
+  explicitly, even when they are identical to the SSL parameters for the Kafka
```
Comment (morsapaes) [inline doc/user/content/release-notes.md:136]: > even when they are identical to the SSL parameters for the Kafka broker

Ah, had a hunch that this was contradicting the docs, but just double checked this seems to have changed there too. That's too bad, it makes the syntax a bit verbose.

File: doc/user/layouts/partials/specifying-aws-credentials.html (hunk 1 above)
Comment (benesch) [reply doc/user/layouts/partials/specifying-aws-credentials.html:16]: Ugh! https://github.com/MaterializeInc/materialize/pull/10658

File: doc/user/content/release-notes.md (hunk 2 above)
Comment (benesch) [reply doc/user/content/release-notes.md:136]: Yep, totally, some discussion on that point here: https://github.com/MaterializeInc/materialize/pull/10381#issuecomment-1029540740. The tl;dr is we have a plan for removing all of the duplication from `CREATE SOURCE`, so didn't want to spend time removing just some of the duplication.

I'm working on a design doc for this `CREATE CONNECTOR` (see also https://github.com/MaterializeInc/materialize/issues/8372; apparently I've been talking about this for a while 🙈 ).

Comment (morsapaes) [reply doc/user/content/release-notes.md:136]: Oh cool, I did not know about `CREATE CONNECTOR`!

**Pull request merged**